src/importexport/capella/tests/capella_tests.cpp

```diff
@@ -123,16 +123,16 @@ TEST_F(Capella_Tests, capxTest3) {
 }
 TEST_F(Capella_Tests, capxTest4) {
     capxReadTest("test4");
-}                                                                 // wrong enharmonic spelling
+}                                                               // wrong enharmonic spelling
 TEST_F(Capella_Tests, capxTest5) {
     capxReadTest("test5");
 }
 TEST_F(Capella_Tests, capxTest6) {
     capxReadTest("test6");
 }
-TEST_F(Capella_Tests, DISABLED_capxTest7) { // wrong cap xml ( element double close (<head pitch="E5"/> </head>) )  // double bar missing (auto-generated by Capella programs)
```

Comment:
That 1st comment apparently is nonsense, there is no such double close in the capx file. Maybe it had been fixed long ago?